src/routes/webshop/cart/+page.server.ts

```diff
@@ -0,0 +1,38 @@
+import prisma from "$lib/utils/prisma";
+import type { PageServerLoad } from "./$types";
+
+export const load: PageServerLoad = async ({ locals }) => {
```

Comment:
On a quick glance this looks like a duplicate of webshop/+page.server.ts. If it is, make the webshop/ one a +layout.server.ts file and let the data trickle down.